Mithril update 2023-10-04 (#263)
input-output-hk/cardano-updates: blog/2023-10-04-mithril.md

---
title: Mithril Team Update
slug: 2023-10-04-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> The Mithril team has redacted a proposal for implementing Mithril in a light wallet: a discussion [Implement Mithril in a light wallet](https://github.com/input-output-hk/mithril/discussions/1273) has been created in order to receive feedback and contributions from the community.

Reviewer: ```suggestion
This week, the Mithril team updated a proposal for implementing Mithril in a light wallet and created a discussion [on implementing Mithril in a light wallet](https://github.com/input-output-hk/mithril/discussions/1273) to receive feedback and contributions from the community.
```

> They completed the PoC to run a Mithril client in a browser with WASM. They also worked on enhancing the explorer UI with SPO tickers that are displayed next to the pool ids. Additionally, they completed a tool for benchmarking the aggregator performances over a range of signers and clients load, and for visualizing the results of these benchmarks.

Reviewer: ```suggestion
They also completed a proof of concept to run a Mithril client in a browser with WASM and worked on enhancing the explorer UI with SPO tickers displayed next to the pool IDs. Additionally, the team finalized a tool for benchmarking aggregator performance over a range of signer and client loads and for visualizing the results of these benchmarks.
```

> Finally, the team added a root page to the aggregator endpoint, they created a status page for the Mithril networks, and they added a section about KES keys rotation in the signer documentation for SPOs.

Reviewer: ```suggestion
Finally, the team added a root page to the aggregator endpoint, created a status page for the Mithril networks, and added a section about KES key rotation in the signer documentation for SPOs.
```
